beetmoverscript/script.py

```diff
@@ -263,6 +310,8 @@ def list_bucket_objects(context, s3_resource, prefix):
     'push-to-candidates': push_to_nightly,
     'push-to-releases': push_to_releases,
     'push-to-maven': push_to_maven,
+    'push-to-nightly-2': artifactMap_push_to_nightly,
```

Comment:
I'd like a clearer name, too. The action is currently an instruction, though, and I don't know if it's a good idea to encode _how_ that action is performed in its name. I suppose the other alternative is to work it out - if the artifact map is present then use it, otherwise fall back.

Reply:
Hm, I like this idea. So let me get this straight. Are you suggesting we should keep the existing action scopes, even when things are turned on in production and toggle between behaviors depending on `artifactMap` being present in the payload or not? 

Also, ++ on not including the `how` into the instruction function name. I think that's a much cleaner approach.